Source/WebCore/Modules/compression/DecompressionStreamDecoder.h

```diff
@@ -25,6 +25,10 @@
 #pragma once
 
 #include "BufferSource.h"
+#ifdef __APPLE__
```

Comment:
We do not use __APPLE__ a lot. Can we use #if PLATFORM(COCOA) instead?

Reply:
Ah, was not aware of that. Going to update this.

Reply:
There are still some __APPLE__ in the current PR